webpack.config.js

```diff
@@ -32,12 +31,9 @@ module.exports = {
         include: [path.resolve(__dirname, "src")],
         use: [
           {
-            loader: "ts-loader",
+            loader: "babel-loader",
             options: {
-              transpileOnly: true,
-              getCustomTransformers: () => ({
-                before: PRODUCTION ? [] : [ReactRefreshTypeScript()],
-              }),
+              plugins: PRODUCTION ? [] : ["react-refresh/babel"],
```

Comment:
However `react-refresh-typescript` doesn't support `commonjs` module, so it's switched to `babel-loader`.